src/bundle/Resources/config/services.yml

```diff
@@ -76,3 +76,7 @@ services:
     EzSystems\HybridPlatformUi\Repository\DecoratedLocationService:
         arguments:
             - "@ezpublish.api.service.location"
+
+    EzSystems\HybridPlatformUi\Repository\ContentTypeService:
+        arguments:
+            - "@ezpublish.signalslot.service.content_type"
```

Comment:
They both map to the same thing but I presume the one you mention is more flexible and is the one to use 👍